### ISSUE TITLE
Large font at launch then smaller after minimizing window

### ISSUE THREAD

**Issue by Emurikku** [Heroic-Games-Launcher/HeroicGamesLauncher#3594]
### Describe the bug

Hello,

First, thanks for the latest version. I have a much better time installing games from GOG.

When go to desktop mode on my Steam Deck, Heroic is launched automatically, minimized, like Steam does. When I click on the system tray icon, the font is very large, as I set it in “Accessibility”. But when I close or minimize the window, the font becomes smaller, as if the accessibility options were taken in account only once.

### Add logs

```text
.
```

### Steps to reproduce

1. Increase font size in “Accessibility”;
2. Launch Heroic;
3. Display window;
4. Minimize;
5. Maximize: font is smaller.

### Expected behavior

The font size should be consistent.

### Heroic Version

Latest Stable

### System Information

SteamOS, Stable branch

**Comment by arielj** [comment]
Closing as a duplicate of https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/3418
